presto-hive/src/main/java/com/facebook/presto/hive/metastore/HivePrivilegeInfo.java

```diff
@@ -103,6 +104,23 @@ public boolean isContainedIn(HivePrivilegeInfo hivePrivilegeInfo)
                         (!isGrantOption() && hivePrivilegeInfo.isGrantOption())));
     }
 
+    public PrivilegeInfo toPrivilegeInfo()
+    {
+        switch (getHivePrivilege()) {
+            case SELECT:
+                return new PrivilegeInfo(Privilege.SELECT, isGrantOption());
+            case INSERT:
+                return new PrivilegeInfo(Privilege.INSERT, isGrantOption());
+            case DELETE:
+                return new PrivilegeInfo(Privilege.DELETE, isGrantOption());
+            case UPDATE:
+                return new PrivilegeInfo(Privilege.UPDATE, isGrantOption());
+            case OWNERSHIP:
```

Comment:
This probably corresponds to all of the privileges. So this method needs to return a list